### DISCUSSION TITLE
Git student Renewal

### DISCUSSION THREAD

**Discussion by f-magdiel** [community/community#28908] [GitHub Education]
Days ago I renewed my git pack for students but the expiration date still hasn't changed. So I was wondering if my git student status is still pending or do I have to wait for the expiration date to come, for it to update.

**Comment by pasindu651** [comment]
Hi @f-magdiel,
It may take a few days for the expiry date to update upon renewal. If not, your application is likely to be quickly renewed if you use a recognizable school-issued email address. Good luck!

**Comment by f-magdiel** [comment]
Hello @pasindu651 , in fact I am using an institutional email from the university

**Reply by pasindu651** [reply]
Hi @f-magdiel, in that case, I'm sure that your student pack will be renewed in no time! Best of luck!

**Comment by f-magdiel** [comment]
Hello, @pasindu651 , there will be no way of knowing that, it is that it has me worried?
